### DISCUSSION TITLE
Keep track of applied commits

### DISCUSSION THREAD

**Discussion by gerardgorrion** [fluxcd/flux2#1257] [Q&A]
Good morning,

We are starting to migrate to flux2 from flux1, and we want the way to tag the commit that kustomize is using, like command "--git-tag" used in fluxctl in flux1 deploy.

These feature is very usefull and currently is a problem that stopped our migration...

We try to "--label" the "flux create source git" deploy, because we don't want to use bootstrap installation mode.

Thanks to all!

**Comment by stefanprodan** [comment]
Flux 2 supports syncing with a Git tag see: https://toolkit.fluxcd.io/components/source/gitrepositories/#checkout-strategies

**Reply by gerardgorrion** [reply]
We don't want to sync, we want to tag currently commit of our repo, to check on every repo what git commit is the currently applied.

**Reply by hiddeco** [reply]
I do not think we are going to bring this behavior back, it relied on periodically overwriting a tag in Git that should be static, and was in my opinion thus an anti-pattern.

If you want to know if a commit has been processed by Flux, the notification-controller has support for reporting back data about  this using Git platform their commit status APIs, see: https://toolkit.fluxcd.io/guides/notifications/#git-commit-status

**Reply by gerardgorrion** [reply]
And this commit status is usable to multipleclusters that attack same bitbucket repo?

**Reply by gerardgorrion** [reply]
We are very dissapointed, this status value not seem to do same function. Flux 1 is in maintenence mode, and flux2 lose this usefull feature.
